dist/docker-images/ziti-router/README.md

```diff
@@ -65,30 +72,72 @@ services:
       ziti-router:
         condition: service_healthy
     command: wget --output-document=- http://hello.internal/
+EOF
+```
 
-  # link the router to the quickstart network so it can reach the Ziti controller
+Your `compose.tproxy.yml` should look like this.
+
+```yaml
```

Comment:
provide a plain sample showing what it should look like, sans comments to conserve visual noise